package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "commitplease",
-  "version": "2.3.1",
+  "version": "2.4.0",
```

Comment:
Gotcha, but then it shouldn't show up in this PR. Seems like something went wrong with the rebase.